examples/stm32f4/src/bin/multiprio.rs

```diff
@@ -62,11 +62,11 @@ use defmt_rtt as _; // global logger
 use panic_probe as _;
 
 use cortex_m_rt::entry;
-use embassy::executor::{Executor, InterruptExecutor};
-use embassy::interrupt::InterruptExt;
 use embassy::time::{Duration, Instant, Timer};
 use embassy::util::Forever;
+use embassy_cortex_m::executor::{Executor, InterruptExecutor};
```

Comment:
Could it make sense to reexport these in embassy-stm32/embassy-nrf/embassy-rp to avoid having to add the embassy-cortex-m to dependencies?

Reply:
Nice idea, done!